compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -177,6 +177,28 @@ class ResolverImpl(
             }
     }
 
+    override fun getFunctionDeclarationsByName(name: KSName, includeTopLevel: Boolean): Sequence<KSFunctionDeclaration> {
+        val qualifier = name.getQualifier()
+        val functionName = name.getShortName()
+        val nonTopLevelResult = this.getClassDeclarationByName(qualifier)?.getAllFunctions()?.filter{ it.simpleName.asString() == functionName }?.asSequence() ?: emptySequence()
+        return if (!includeTopLevel) nonTopLevelResult else {
+            nonTopLevelResult.plus(module.getPackage(FqName(qualifier))
+                .memberScope.getContributedDescriptors(DescriptorKindFilter.FUNCTIONS) { it.asString() == functionName }
+                .filterIsInstance<MemberDescriptor>().mapNotNull {  it.toKSDeclaration() as? KSFunctionDeclaration} )
+        }
+    }
+
+    override fun getPropertyDeclarationByName(name: KSName, includeTopLevel: Boolean): KSPropertyDeclaration? {
+        val qualifier = name.getQualifier()
+        val propertyName = name.getShortName()
+        val nonTopLevelResult = this.getClassDeclarationByName(qualifier)?.getAllProperties()?.singleOrNull { it.simpleName.asString() == propertyName }
```

Comment:
Same for `getAllProperties()`; Inherited properties might not be needed / desired.